### DISCUSSION TITLE
Dockerized Netbox behind Traefik Reverse Proxy

### DISCUSSION THREAD

**Discussion by unnuetz** [netbox-community/netbox#7883] [Q&A]
Hi Lovely Community,

i am facing an issue, what is to me currently a bit unclear why this is happening.

I have a Netbox Container behind a traefik reverse Proxy to handle https for netbox.

It is working problery.

What i customized is some css adjustments and replaced the default netbox logo using:

docker-compose exec -T --user root netbox tar x -zvf - -C /opt/netbox/netbox/static/img < logo.tar.gz
docker-compose exec -T --user root netbox tar x -zvf - -C /opt/netbox/netbox/static/bootstrap-3.4.1-dist/css < bootstrap.min.css.tar.gz
docker-compose exec -T --user root netbox tar x -zvf - -C /opt/netbox/netbox/static/css < base.tgz

Now i want to access netbox from a host **which has no access to the internet** and only access to the traefik ip with its port 443.

When i launch a webbrowser an open the netbox url i am receiving the following error:

![image](https://user-images.githubusercontent.com/25468412/142616307-513208f2-dd43-4d98-9f15-c3baecb5807d.png)

If i am opening the url from a pc which has access to the "internet" the page is completly loading proberly.

The https cert ca i am using, is internally windows domain root ca, so i am not using a letsencrypt or something like that.

![image](https://user-images.githubusercontent.com/25468412/142616980-5fa0e7db-73d1-4bed-8fd8-77fcb2df397d.png)

Static Media Failure
The following static media file failed to load:
bootstrap-3.4.1-dist/css/bootstrap.min.css

Check the following:
manage.py collectstatic was run during the most recent upgrade. This installs the most recent iteration of each static file into the static root path.
...

I could solve the issue by using the command inside the netbox container:
manage.py collectstatic

but once i did that, all my customizations like color css and logo is gone/back to default.

What did i miss here?

netbox version 2.11.12

**Comment by candlerb** [comment]
Permissions maybe?

**Comment by kkthxbye-code** [comment]
Modifying a running docker image is perhaps the worst way you could have chosen to customize netbox.

If you really want to change the look of your netbox instance, you can either fork the netbox version you are using (seems to be old) and build your own images using build.sh with the SRC_ORG and SRC_REPO options. Alternatively you could try to see if your modifications are possible with the BANNER_TOP/BANNER_BOTTOM options. HTML is allowed, so it might be possible to overwrite stuff with a <style> block or some javascript.

https://netbox.readthedocs.io/en/stable/configuration/optional-settings/#banner_top

>If i am opening the url from a pc which has access to the "internet" the page is completly loading proberly.

Probably not, the files are probably just cached on that machine.
